### PR TITLE
js_task_generate_table_DOM

### PR DESCRIPTION
**Pull request by mykolahuryn**

### CODE REVIEW

**Comment by mykolahuryn** [comment]
[DEMO LINK](https://mykolahuryn.github.io/js_task_generate_table_DOM/)

File: src/scripts/main.js [hunk 1]
```diff
@@ -2,7 +2,22 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+people.map(person => {
+  person.age = person.died - person.born;
+  person.century = Math.ceil(person.died / 100);
+});
```
Comment (agniya-i) [inline src/scripts/main.js:8]: better to do it right in next `for`.

it's not a good idea to change input data, also you should set the results of `.map` to a variable if you want to use it

File: src/scripts/main.js [hunk 2]
```diff
@@ -2,7 +2,22 @@
...
+
+for (const person of people) {
+  table.insertAdjacentHTML('beforeend', `
+        <tr>
+            <td>${person.name}</td>
+            <td>${person.sex === 'm' ? 'Male' : 'Female'}</td>
+            <td>${person.born}</td>
+            <td>${person.died}</td>
+            <td>${person.age}</td>
+            <td>${person.century}</td>
```
Comment (agniya-i) [inline src/scripts/main.js:20]:
```suggestion
            <td>${person.died - person.born}</td>
            <td>${Math.ceil(person.died / 100)}</td>
```

**Review by agniya-i — CHANGES_REQUESTED** [review]

File: src/scripts/main.js [hunk 3]
```diff
@@ -2,7 +2,17 @@
...
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('table').children[0];
 
-// write your code here
+for (const person of people) {
+  table.insertAdjacentHTML('beforeend', `
+        <tr>
+            <td>${person.name}</td>
```
Comment (Anastasiia-Svintsova) [inline src/scripts/main.js:10]: Would be good to destructure the person

**Review by Anastasiia-Svintsova — APPROVED** [review]

**Pull request closed without merging**